crema/confidence.py

```diff
@@ -17,6 +21,7 @@ def assign_confidence(
     desc=None,
     eval_fdr=0.01,
     method="tdc",
+    pep_fdr_type="psm-peptide",
```

Comment:
Yes, it will match the Crema paper. The original name (in the paper describing it) it was called "psm-and-peptide". In the Crema paper I decided to shorten it to this.